package.json

```diff
@@ -4,18 +4,19 @@
   "description": "react tooltip component",
   "main": "dist/index.js",
   "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
+  "jsnext:main": "dist/index.ex.js",
```

Comment:
index.ex.js is not generated in build, why is this change for?

Reply:
Sorry, that was a typo. Fixed.